Core/Services/TripService.cs

```diff
@@ -58,12 +56,17 @@ public async Task CreateTripAsync(CreateTripDTO createTripDTO, string creatorId)
             trip.IsEnded = false;
```

Comment:
is the default value of this field "true"?

Reply:
No, false